integrations/tensorflow/iree-dialects/test/iree_linalgext/invalid.mlir

```diff
@@ -107,16 +107,16 @@ func @scatter_mixed_tensor_memref(
 
 func @scatter_mixed_tensor_memref(
     %update : tensor<?x?xf32>, %indices : tensor<?x1xi32>,
-    %original : tensor<?x?xf32>) -> memref<?x?xf32> {
-  // expected-error @+1 {{expected type of `outs` operand #0 'tensor<?x?xf32>' to be same as result type 'memref<?x?xf32>'}}
+    %original : tensor<?x?xf32>) -> tensor<4x?xf32> {
```

Comment:
Why do we need this change? Based on what the test says, it is explicitly checking for illegality of mixed tensor and memref. Was there some other error on this?

Reply:
There is some new verifier that checks that this op can only return tensors. The verification seemed to be about type mismatch and not specifically about tensor vs memref. What do you recommend there?

Reply:
Ok, its fine then. Maybe change the name from scatter_mixed_tensor_memref -> scatter_result_type_mismatch.

Reply:
sure